### DISCUSSION TITLE
Global Values per Section

### DISCUSSION THREAD

**Discussion by alexanderloewe** [craftcms/cms#12580] [Ideas]
In Craft CMS 5, the global values are migrated to Entries/Section. In that context it would be fitting if you could assign some kind of global values to a specific section.

What I mean by this are global values which could be assigned to a specific Section. For example in the section Blog:

Fields in which you can assign values that are specific to the blog section (Chanel).

**Comment by thupsi** [comment]
Hi! 
I have opened two relevant discussions in the past. You can see them here:
https://github.com/craftcms/cms/discussions/9736
https://github.com/craftcms/cms/discussions/9783

I _think_  (and hope) that the **Unified Content View** coming in Craft 5 will provide some functionality in this general direction, but details are still sparse. Since Craft 5 is all about content modelling, I'll keep my fingers crossed 🤞
